src/JsonLd/ContextBuilderInterface.php

```diff
@@ -53,4 +52,905 @@ public function getResourceContext(string $resourceClass, int $referenceType = U
      * Gets the URI of the given resource context.
      */
     public function getResourceContextUri(string $resourceClass, int $referenceType = UrlGeneratorInterface::ABS_PATH): string;
+
+    public const HYDRA_CONTEXT = [
+        '@context' => [
```

Comment:
Maybe could we store this in an external PHP file that we automatically generate from the upstream JSON (so we can regenerate the file if needed).